python/py-tensorflow/Portfile

```diff
@@ -94,17 +94,19 @@ if {${name} ne ${subport}} {
         port:py${python.version}-future \
         port:py${python.version}-pip \
         port:py${python.version}-mock \
-        port:bazel-2.0 \
+        port:bazel \
```

Comment:
Are you sure this works ? Tensorflow has historically not been able to be built with the latest bazel versions.

Reply:
Well it at least does not build with bazel 2 anymore; see https://trac.macports.org/ticket/61259